android/vpnClient/src/main/java/org/mozilla/firefox/vpn/qt/VPNActivity.java

```diff
@@ -162,6 +162,7 @@ private void initServiceConnection(){
   private final int ACTION_NOTIFICATION_PROMPT_SENT = 19;
   private final int EVENT_PERMISSION_REQURED = 6;
   private final int EVENT_DISCONNECTED = 2;
+  private final int EVENT_CANCELLED = 9;
```

Comment:
`cancelled` for this (in several files) almost makes it sound like the user cancelled something. Did you consider something like `EVENT_ONBOARDING_COMPLETE` (that's a little long, but hopefully you get my gist)?

Reply:
my original reasoning was to make the naming for all of this generic enough so that it made sense if the reasoning was something other than onboarding in the future. as you can see, I eventually went against that on the iOS side, so I will make it more consistent on Android as well